src/tikz_export.jl

```diff
@@ -0,0 +1,153 @@
+export export_performance_profile_tikz
+
+"""
+    function export_performance_profile_tikz(T, filename; kwargs...)
+
+Export tikz figure of the performance profiles given by `T` in `filename`.
+
+## Arguments
+
+* `T :: Matrix{Float64}`: each column of `T` defines the performance data for a solver (smaller is better).
+  Failures on a given problem are represented by a negative value, an infinite value, or `NaN`.
+* `filename :: String` : path to the tikz exported file.
+
+## Keyword Arguments
+
+* `solvernames :: Vector{String} = []` : names of the solvers, should have as many elements as the number of columns of `T`. If empty, use the labels returned by `performance_profile_axis_labels`.
+* `xlim::AbstractFloat=10.` : size of the figure along the x axis. /!\\ the legend is added on the right hand side of the figure.
+* `ylim::AbstractFloat=10.` : size of the figure along the y axis.
+* `nxgrad::Int=5` : number of graduations on the x axis.
+* `nygrad::Int=5` : number of graduations on the y axis.
+* `grid::Bool=true` : display grid if true.
+* `colours::Vector{String} = []` : colours of the plots, should have as many elements as the number of columns of `T`.
+* `linestyles::Vector{String} = []` : line style (dashed, dotted, ...) of the plots, should have as many elements as the number of columns of `T`.
+* `linewidth::AbstractFloat = 1.0` : line with of the plots.
+* `xlabel::String = ""` : x axis label. If empty, uses the one returns by `performance_profile_axis_labels`.
+* `ylabel::String = ""` : x axis label. If empty, uses the one returns by `performance_profile_axis_labels`.
+
+Other keyword arguments are passed `performance_profile_data`.
+
+"""
+function export_performance_profile_tikz(
+  T::Matrix{Float64},
+  filename::String;
+  solvernames::Vector{String}=String[],
+  xlim::AbstractFloat=10.,
+  ylim::AbstractFloat=10.,
+  nxgrad::Int=5,
+  nygrad::Int=5,
+  grid::Bool=true,
+  # markers::Vector{S} = String[],
+  colours::Vector{String} = String[],
+  linestyles::Vector{String} = String[],
+  linewidth::AbstractFloat = 1.0,
+  xlabel::String = "",
+  ylabel::String = "",
+  kwargs...)
+
+
+
+  logscale = true
+  if haskey(kwargs,:logscale)
+    logscale = kwargs[:logscale]
+  end
+  xlabel_def, ylabel_def, solvernames = performance_profile_axis_labels(solvernames, size(T, 2), logscale; kwargs...)
+  isempty(xlabel) && (xlabel=xlabel_def)
+  isempty(ylabel) && (ylabel=ylabel_def)
+
+  # some offsets
+  axis_tik_l = 0.2 # tick length
+  lgd_offset = 0.5 # space between figure and legend box
+  lgd_box_length = 3. # legend box length
+  lgd_plot_length = 0.7 # legend plot length
+  lgd_v_offset = 0.7 # vertical space between legend items 
+
+  label_val = [0.2,0.25,0.5,1] # possible graduation labels along axes are multiples of label_val elements times 10^n
+  ymax = 1.0
```

Comment:
Should these be keyword arguments?

Reply:
They can be indeed, I suppose it's better to give the user as much freedom as possible.